code/__DEFINES/dcs/signals/signals_atom/signals_atom_attack.dm

```diff
@@ -6,6 +6,8 @@
 #define COMSIG_PARENT_ATTACKBY "atom_attackby"
 /// From base of [atom/proc/attacby_secondary()]: (/obj/item/weapon, /mob/user, params)
 #define COMSIG_PARENT_ATTACKBY_SECONDARY "atom_attackby_secondary"
+///from [/item/afterattack()] called on the item which was attacked: (/obj/item/weapon, /mob/user, proximity_flag, click_parameters)
```

Comment:
dont you call it on the target though?

Reply:
This comment could definitely be clearer about what I mean